Extensions/BBTextLabel/JsExtension.js

```diff
@@ -0,0 +1,385 @@
+/**
+ * This is a declaration of an extension for GDevelop 5.
+ *
+ * ℹ️ Run `node import-GDJS-Runtime.js` (in newIDE/app/scripts) if you make any change
+ * to this extension file or to any other *.js file that you reference inside.
+ *
+ * The file must be named "JsExtension.js", otherwise GDevelop won't load it.
+ * ⚠️ If you make a change and the extension is not loaded, open the developer console
+ * and search for any errors.
+ *
+ * More information on https://github.com/4ian/GDevelop/blob/master/newIDE/README-extensions.md
+ */
+
+//TODO: add base color, base font, base size
+const MultiStyleText = require('./dist/pixi-multistyle-text.umd');
+console.log(MultiStyleText);
+module.exports = {
+  createExtension: function(_, gd) {
+    const extension = new gd.PlatformExtension();
+    extension
+      .setExtensionInformation(
+        'BBText',
```

Comment:
nitpicking, but you should name your extension consistently with the name of the folder: BBTextLabel

Reply:
In fact looking at the rest of the extension, it's the folder that should be named just "BBText", as the extension.
"Label" does not provide anything more than text I think.

So it should be either:
`BBText` or `BBTextObject` for the extension name.
I would suggest `BBTextObject` for the extension name, as other extensions providing a single object are also named SomethingObject.
(you would name it "BBText" if the extension was providing multiple objects or behaviors, or both).